tests/kernel/run_ltp.pm

```diff
@@ -366,7 +368,9 @@ sub run {
     }
 
     if (is_serial_terminal) {
-        script_run($klog_stamp);
+        script_run($klog_stamp_base . "kmsg");
+        # SLE11-SP4 don't support ignore_loglevel, due that stamp is not printed in console
+        script_run($klog_stamp_base . $serialdev) if is_sle('<12');
```

Comment:
That is some seriously unreadable piece of code. This would be better:

    my $klog_stamp = "OpenQA::run_ltp.pm: Starting $test->{name}";

    script_run("echo '$klog_stamp' > /dev/kmsg");
    script_run("echo '$klog_stamp' > /dev/$serialdev") if is_sle('<12');